app/src/main/java/mozilla/lockbox/view/DialogFragment.kt

```diff
@@ -7,12 +7,12 @@
 package mozilla.lockbox.view
 
 import android.os.Bundle
-import android.support.annotation.StringRes
-import android.support.v4.app.DialogFragment
+import androidx.annotation.StringRes
+import androidx.fragment.app.DialogFragment
 import android.view.View
 import mozilla.lockbox.flux.Presenter
 
-open class DialogFragment : DialogFragment() {
+open class DialogFragment : androidx.fragment.app.DialogFragment() {
```

Comment:
nit: import this rather than having the long class name

Reply:
:+1: